jcl/j9modules.xml

```diff
@@ -47,11 +47,12 @@
 				</delete>
 			</case>
 			<case value="java.base">
-				<removeexport package="com.ibm.cuda" />
-				<removeexport package="com.ibm.dataaccess" />
-				<removeexport package="com.ibm.jvm" />
-				<removeexport package="com.ibm.oti.shared" />
-				<removeexport package="com.ibm.sharedclasses.spi" />
+				<removeexportsoropens action="exports" package="com.ibm.cuda" />
+				<removeexportsoropens action="exports" package="com.ibm.dataaccess" />
+				<removeexportsoropens action="exports" package="com.ibm.jvm" />
+				<removeexportsoropens action="exports" package="com.ibm.oti.shared" />
+				<removeexportsoropens action="exports" package="com.ibm.sharedclasses.spi" />
+				<replaceexportsoropens action="exports" module.old="com.ibm.dtfj" module.new="" />
```

Comment:
Why is module.new empty?

Reply:
The macro allows `module.new` to be empty, which simply removes the old module name from the export line, or the entire line if it's the only module being exported to.

Reply:
What I don't understand is why the exports to dtfj are removed.

Reply:
Because we are merging with the latest module-info.java files which adds back the exports to openj9.dtfj?

Reply:
We need to remove references to the old module name `com.ibm.dtfj` or else it will look for and fail to find `com.ibm.dtfj` when recompiling the module in the compose step. I don't think it makes a difference whether we remove the name or replace it with `openj9.dtfj`, since the merging we do takes care of duplicates.